### ISSUE TITLE
Need usage report

### ISSUE THREAD

**Issue by scarlett2018** [Microsoft/pai#2127]
Job Summary, User Usage Summary, VC Usage Summary.

By time, by status

**Comment by xudifsd** [comment]
These info can be get from launcher from `:9086/v1/Frameworks`, if combined info from #2073 , we can calculated resource that wasted(the job finally failed).

**Comment by scarlett2018** [comment]
> These info can be get from launcher from `:9086/v1/Frameworks`, if combined info from #2073 , we can calculated resource that wasted(the job finally failed/killed).

Would you like to merge #2073 with this item? what's the estimation to have both #2127 and #2073 in place? Let's combine them if it makes tracking easier.

**Comment by xudifsd** [comment]
No, I think #2073 is relatively easy to implement, but this requires more efforts. Let's track them in different issues.

**Comment by scarlett2018** [comment]
There are some experiment for job dashboard done in PowerBI, it's time to revisit, whether these things work well for v1.x. And whether there are any new needs to better understand the overall job utilization.
